pagopa/io-app
# feat: [IOBP-636,IOBP-646] Add legacy transactions list

## ⚠️ This PR depends on #5768 ⚠️

## Short description
This PR implements the ability to view “legacy” transactions that are not available through biz-events at the moment. 

## List of changes proposed in this pull request
- Added a legacy list screen that retrieves the old transactions not available with biz-events;
- Connected the legacy list screen with the previous transaction details that shows transaction info with a reduced amount of data;
- Added a CTA to access to this new page from:
   - `PaymentsHomeEmptyScreenContent`
   - Into the biz-events transaction list when the user reaches the end of the list.

## How to test
- Checkout the dev-server with the following PR: https://github.com/pagopa/io-dev-api-server/pull/369
- Open the new payment landing page and try to reach the CTA from the entry point listed above, you should be able to see the previous transactions with less data shown.

## Preview

https://github.com/pagopa/io-app/assets/34343582/7e759311-d0f6-48af-962e-edb28c0bb682
